.adr-dir

```diff
@@ -0,0 +1 @@
+doc/decisions
```

Comment:
I don't like how this pollutes the project root directory and will take care of it by providing a separate script to use [`adr-tools`](https://github.com/npryce/adr-tools) later on.

Reply:
I like to keep the root directory of a project as clean as possible so contributors can quickly orient themselves. Adding an extra `.adr-dir` file that is likely not relevant to a good chunk of contributors goes against that.

Reply:
Why do we need the adr tool exactly? Why not just files in a directly? We already have a pretty interface with GitHub.

Reply:
`adr-tools` provides some neat helpers but it's not important for now. I deleted the `.adr-dir` file and will introduce `adr-tools` separately.